src/main/java/com/nutomic/syncthingandroid/syncthing/AppConfigReceiver.java

```diff
@@ -0,0 +1,101 @@
+package com.nutomic.syncthingandroid.syncthing;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.support.v4.app.NotificationCompat;
+
+import com.nutomic.syncthingandroid.R;
+import com.nutomic.syncthingandroid.activities.MainActivity;
+
+/**
+ * Broadcast-receiver to control and configure SyncThing remotely
+ *
+ * Created by sqrt-1764 on 25.03.16.
+ */
+public class AppConfigReceiver extends BroadcastReceiver {
+    private static final int ID_NOTIFICATION_BACKGROUND_ACTIVE = 2;
+
+    public static final String ACTION_SYNCTHINGSERVIVE = "syncthingservice";
+
+    @Override
+    public void onReceive(Context context, Intent intent) {
+        final Bundle extras = intent.getExtras();
+
+        if (extras == null) return;                         // Nothing to do or to configure
+
+        if (extras.containsKey(ACTION_SYNCTHINGSERVIVE)) {
+            String value = extras.getString(ACTION_SYNCTHINGSERVIVE);
+            if (value != null) {
+                value = value.toLowerCase();
+
+                switch (value) {
+                    case "start":
+                        context.startService(new Intent(context, SyncthingService.class));
+                        break;
+
+                    case "stop":
+                        if (SyncthingService.alwaysRunInBackground(context)) {
+                            final String msg = context.getString(R.string.appconfig_receiver_background_enabled);
+
+                            NotificationCompat.Builder nb = new NotificationCompat.Builder(context)
+                                    .setWhen(System.currentTimeMillis())
+                                    .setContentText(msg)
+                                    .setTicker(msg)
+                                    .setStyle(new NotificationCompat.BigTextStyle().bigText(msg))
+                                    .setContentTitle(context.getText(context.getApplicationInfo().labelRes))
+                                    .setSmallIcon(context.getApplicationInfo().icon)
+                                    .setAutoCancel(true)
+                                    .setContentIntent(PendingIntent.getActivity(context, 0, new Intent(context, MainActivity.class), PendingIntent.FLAG_UPDATE_CURRENT));
```

Comment:
I just copied and pasted a piece of code I used for my other notifications.

SetTicker used to show a text when the notification first was displayed. This behaviour was changed later. But it still seems to be useful:
http://stackoverflow.com/questions/29725102/is-notification-builder-setticker-still-useful-in-android-5-and-above

Regarding setWhen and setAutocancel:
The documentation does not mention the default behaviour, so I explicitly did set those values to be on the safe side.
But I have no problem to remove those calls.